.env.template

```
@@ -1,2 +1,2 @@
 NEMLI__DISCORD__TOKEN=
-NEMLI__DISCORD__PREFIX=
+NEMLI__OPENAI__API__KEY=
```

```suggestion
NEMLI__OPENAI__API_KEY=
```

`api_key` deve ser uma propriedade dentro de um objeto `OpenApi(BaseModel). Assim, o namespacing vem automático e você fica _com um código bem aesthetic, nengue._

---

Done ✅

---

Done ✅